src/index.html

```diff
@@ -7,8 +7,54 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#Home" class="logo">
+          <img
+            src="images/logo.png"
+            alt="logo"
+            class="logo"
+          >
+        </a>
+
+      </div>
+      <nav class="header__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple" class="nav__item is-active">Apple</a>
+          </li>
+          <li class="nav__item">
```

Comment:
```suggestion
          <li class="nav__item">
            <a href="#Apple" class="nav__item is-active">
              Apple
            </a>
          </li>
          
          <li class="nav__item">
```
fix code style all over the page